scripts/util/reject-previous-pipeline-executions.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const AWS = require('aws-sdk');
+const { filter, flatMap, flow, map } = require('lodash/fp');
```

Comment:
GLOBAL NIT: Do we need lodash here? I've found projects with `lodash/fp` a little harder to read, maintain, and get others involved with over time (I used to try and use it more but gave up...) Most of the stuff here seems we can just do real ES instead?

Reply:
Yeah I always waffle back and forth on `lodash/fp`, so I think it makes sense to remove here. May just want to include normal `flatMap` still!

Reply:
Ticket in case I can't get to this yet: https://github.com/FormidableLabs/badge-matrix-next/issues/15